internal/services/instances.go

```diff
@@ -117,3 +111,93 @@ func (s *ServiceClient) Create(ctx context.Context, namespace, name string, cata
 		ctx, unstructureHelmChart, metav1.CreateOptions{})
 	return errors.Wrap(err, "error creating helm chart")
 }
+
+func (s *ServiceClient) List(ctx context.Context, namespace string) ([]*models.Service, error) {
+	serviceList := []*models.Service{}
+
+	unstructuredServiceList, err := s.helmChartsKubeClient.Namespace(helmchart.Namespace()).List(ctx, metav1.ListOptions{})
```

Comment:
You can use a label selector here to filter only the helmcharts that have the label key. Simply setting the label key, checks for existence of that label (with no `=`). The `kubectl` equivalent:

```
kubectl get helmcharts -A -l "application.epinio.io/catalog-service-name"
```